Detectors/MUON/MID/Clustering/include/MIDClustering/PreClustersDE.h

```diff
@@ -80,9 +79,9 @@ class PreClustersDE
   void reset();
 
  private:
-  int mDEId = 99;                                ///< Detection element ID
```

Comment:
Frankly, I would prefer `= 99` and not {99}. Also, what is the point of adding {}? That is implied for containers because they have a default constructor.

Reply:
Well, in the coding convention the = sign is preferred but the brace initialisation is also possible. Anyways I changed back to assignment. Also, I removed the {}. I only keep the initialisation of the functional. 
The commits can be squashed.
Thanks.
Cheers,
Diego